docs/codeql/codeql-cli/creating-codeql-query-suites.rst

```diff
@@ -121,6 +126,14 @@ In addition to metadata tags, the keys in the constraint block can also be:
 Examples
 ~~~~~~~~
 
+To define a suite that selects all queries in the default suite of the
+``codeql/cpp-queries`` QL pack, and then refines them to only include
+security queries, use::
+
+   - queries: codeql/cpp-queries
```

Comment:
Should this be `qlpack` rather than `queries`?

Reply:
Yes. My mistake.

Reply:
```suggestion
To define a suite that selects all queries in the default suite of the
``codeql/cpp-queries`` QL pack, and then refines them to only include
security queries, use::

   - qlpack: codeql/cpp-queries
```